_includes/includes/kb_doc_template.php

```diff
@@ -158,27 +164,48 @@ function write_version_history($pagename){
       }
       $i++;
     }
-    $string.= '</ul><br>';
+    $string.= '</ul>';
+    return $string;
+  }
+
+  /* 
+    This functions works to show the keyboard download link
+    on the keyboard help documentation of help.keyman.com
+  */
+  function download_keyboard($pagename) {
```

Comment:
```suggestion
  function build_download_keyboard_link_html() {
```

We don't use the `$pagename` variable, so don't include it.